src/parser/stream.rs

```diff
@@ -87,51 +87,35 @@ impl<'a> TokenStream<'a> {
     }
 
     /// Get current token.
-    pub fn current(&self) -> &Token {
-        let mut cursor = self.cursor;
-        loop {
-            if cursor >= self.length {
-                return &self.default;
-            }
-
-            let current = &self.tokens[cursor];
-
-            if matches!(
-                current.kind,
-                TokenKind::SingleLineComment(_)
-                    | TokenKind::MultiLineComment(_)
-                    | TokenKind::HashMarkComment(_)
-                    | TokenKind::DocumentComment(_)
-            ) {
-                cursor += 1;
-                continue;
-            }
-
-            return current;
+    pub const fn current(&self) -> &Token {
+        if self.cursor >= self.length {
+            return &self.default;
         }
+
+        &self.tokens[self.cursor]
```

Comment:
we don't need to skip comments here, `::new` skips comments, and so does `next()`, so we know that `current` doesn't refer to a comment.